### PR TITLE
first round of gov audit edits

### PR DESCRIPTION
**Pull request by abbey-titcomb**

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-> ❗ This document describes the current governance processes of Radworks. It is version-controlled and can only updated via off-chain approval.
-
-> 📝 For more on how this document is maintained please refer to contributing.md
+> ❗ This document describes the current governance processes of Radworks. For more on how this document is maintained please refer to [contributing.md](/contributing.md)
 
 
 # Overview
 
-Radworks is governed by on-chain and off-chain governance processes. These together make up Radworks' **governance system** 
+Radworks is governed by on-chain and off-chain governance processes. These processes together make up Radworks' **governance system**. Holders of Radworks' native governance token ($RAD) can participate in governance by creating, voting and discussing RAD Governance Proposals (RGPs).
```
Comment (shelbeeee) [inline manual.md]: maybe change to "creating, discussing and voting on"

File: manual.md [hunk 2]
```diff
@@ -16,27 +14,43 @@ The Radworks Treasury is controlled by a tokenized governance mechanism ([Compou
 Currently, these actions include:
 
 - Treasury distributions of assets (e.g. RAD, ETH, and USDC)
-- Upgrades to the on-chain governance system (Compound Governor)
-- Changes to parameters of Treasury-governed contracts (e.g. Drips)
+- Upgrades to the on-chain governance system (e.g. Governance, Timelock)
```
Comment (shelbeeee) [inline manual.md]: maybe instead of governance we use quorum as the example?

File: manual.md [hunk 3]
```diff
@@ -16,27 +14,43 @@ The Radworks Treasury is controlled by a tokenized governance mechanism ([Compou
...
+- Changes to parameters of Treasury-governed protocols (e.g. Drips)
 
 To learn more about token governance and how Radworks’ on-chain governance system works, [see here](https://wiki.tally.xyz/docs/compound-governor).
 
 ### Off-chain
 
-In addition to the on-chain governance system, Radworks manages a set of off-chain processes and resources that define the rules for participating in the DAO. These processes and resources are maintained off-chain, but updates to these documents still require formal community approval via Snapshot vote. An off-chain review process is also required for all proposals before they can be submitted on-chain.
+In addition to the on-chain governance system, Radworks manages a set of off-chain processes and resources that define the rules for participating in governance. These processes and resources are maintained off-chain, but updates to these documents still require formal community approval via Snapshot vote. An off-chain review process (see [Formal Review](#formal-review) is also required for all proposals before they can be submitted on-chain.
```
Comment (shelbeeee) [inline manual.md:24]: Maybe change to: 
In addition to the on-chain governance system, Radworks utilizes a formal off-chain review process to gain community consensus around making changes to off-chain process and resources, as well as to review proposals intended to be submitted on-chain. See Formal Review (link) below. 

updates to these documents still require community approval via Formal Review (link), including a Snapshot poll. All proposals tha

**Review by shelbeeee — CHANGES_REQUESTED** [review]
few suggestions, otherwise looks good.

File: manual.md (hunk 1 above)
Comment (abbey-titcomb) [inline manual.md]:
```suggestion
Radworks is governed by on-chain and off-chain governance processes. These processes together make up Radworks' **governance system**. Holders of Radworks' native governance token ($RAD) can participate in governance by creating, discussing and voting on RAD Governance Proposals (RGPs).
```

File: manual.md (hunk 2 above)
Comment (abbey-titcomb) [inline manual.md]:
```suggestion
- Upgrades to the on-chain governance system (e.g. Quorum, Timelock)
```

File: manual.md (hunk 3 above)
Comment (abbey-titcomb) [reply manual.md:24]: Did you mean to complete this sentence?

Comment (shelbeeee) [reply manual.md:24]: Shoot sorry not sure what happened there. I don't think that last incomplete sentence was supposed to be there as I tried to summarize the actions in the sentence above.

**Review by shelbeeee — APPROVED** [review]
Lgtm

**Review by shelbeeee — APPROVED** [review]
Lgtm!

**Pull request merged**